### DISCUSSION TITLE
Setting up issues

### DISCUSSION THREAD

**Discussion by abrookbank** [neonbjb/tortoise-tts#385] [Q&A]
I am somewhat new to this I installed python 3.7 and afaik the latest version of pip. I seem to be having an issue with NumPy. I am so lost on what is wrong. I've installed several different programs and Shells which obviously did nothing because I'm missing NumPy but I can't seem to install it. Here is a breakdown of the error I get when I try to add the ./requirements.txt 

I am using Conda in this example, I was following the instructions on the readme

ValueError: 'build\\py3k\\scipy' is not a directory
![scipyerror](https://user-images.githubusercontent.com/30681955/229639911-a0caae26-973f-4633-9564-71fa9198d4a8.jpg)

**Comment by R-Olivaw** [comment]
Have you looked through [this thread](https://github.com/neonbjb/tortoise-tts/issues/281)? Might answer your question if you're on Windows 10 or 11.

**Reply by abrookbank** [reply]
I'm on Windows 11 and I noticed at the end of that thread someone else was having the same issue but no solution has been posted. Something do to with scipy not loading correctly. Anyway to clear it all out and start over?? I'm wondering if there was something that wasn't loaded correctly during the initial install.
